packages/firebase_auth/firebase_auth/windows/firebase_auth_plugin.cpp

```diff
@@ -157,6 +157,8 @@ PigeonUserInfo FirebaseAuthPlugin::ParseUserInfo(
   result.set_photo_url(user->photo_url());
   result.set_provider_id(user->provider_id());
   result.set_uid(user->uid());
+  result.set_creation_timestamp(user->metadata().creation_timestamp);
+  result.set_last_sign_in_timestamp(user->metadata().last_sign_in_timestamp);
```

Comment:
Would these already correctly map to `UserMetadata.creationTime` and `UserMetadata.lastSignInTime`?

Reply:
Yes, exactly 👍